src/System.Runtime/tests/System/Guid.cs

```diff
@@ -3,194 +3,101 @@
-        Assert.True(Guid.TryParseExact("{a8a110d5-fc49-43c5-bf46-802db8f843ff}", "B", out g));
-        Assert.Equal(_testGuid, g);
-        Assert.True(Guid.TryParseExact("(a8a110d5-fc49-43c5-bf46-802db8f843ff)", "P", out g));
-        Assert.Equal(_testGuid, g);
-        Assert.True(Guid.TryParseExact("{0xa8a110d5,0xfc49,0x43c5,{0xbf,0x46,0x80,0x2d,0xb8,0xf8,0x43,0xff}}", "X", out g));
-        Assert.Equal(_testGuid, g);
-
-        Assert.False(Guid.TryParse("a8a110d5fc4943c5bf46802db8f843f", out g)); // One two few digits
-        Assert.False(Guid.TryParseExact("a8a110d5-fc49-43c5-bf46-802db8f843ff", "N", out g)); // Contains '-' when "N" doesn't support those
+    public static void TestCtor_Int_Short_Short_Byte_Byte_Byte_Byte_Byte_Byte_Byte_Byte()
+    {
+        var guid = new Guid(unchecked((int)0xa8a110d5), unchecked((short)0xfc49), 0x43c5, 0xbf, 0x46, 0x80, 0x2d, 0xb8, 0xf8, 0x43, 0xff);
+        Assert.Equal(s_testGuid, guid);
     }
 
     [Fact]
-    public static void TestGetHashCode()
+    public static void TestCtor_UInt_UShort_UShort_Byte_Byte_Byte_Byte_Byte_Byte_Byte_Byte()
     {
-        // Int32 Guid.GetHashCode()
-        Assert.NotEqual(_testGuid.GetHashCode(), Guid.Empty.GetHashCode());
+        var guid = new Guid(0xa8a110d5, 0xfc49, 0x43c5, 0xbf, 0x46, 0x80, 0x2d, 0xb8, 0xf8, 0x43, 0xff);
+        Assert.Equal(s_testGuid, guid);
     }
 
     [Fact]
-    public static void TestToString()
+    public static void TestNewGuid()
     {
-        // String Guid.ToString()
-        // String Guid.ToString(String)
-        // String Guid.System.IFormattable.ToString(String, IFormatProvider) // The IFormatProvider is ignored so don't need to test
+        Guid guid1 = Guid.NewGuid();
+        Assert.NotEqual(Guid.Empty, guid1);
 
-        Assert.Equal(_testGuid.ToString(), "a8a110d5-fc49-43c5-bf46-802db8f843ff");
-        Assert.Equal(_testGuid.ToString("N"), "a8a110d5fc4943c5bf46802db8f843ff");
-        Assert.Equal(_testGuid.ToString("D"), "a8a110d5-fc49-43c5-bf46-802db8f843ff");
-        Assert.Equal(_testGuid.ToString("B"), "{a8a110d5-fc49-43c5-bf46-802db8f843ff}");
-        Assert.Equal(_testGuid.ToString("P"), "(a8a110d5-fc49-43c5-bf46-802db8f843ff)");
-        Assert.Equal(_testGuid.ToString("X"), "{0xa8a110d5,0xfc49,0x43c5,{0xbf,0x46,0x80,0x2d,0xb8,0xf8,0x43,0xff}}");
+        Guid guid2 = Guid.NewGuid();
+        Assert.NotEqual(guid1, guid2);
     }
 
     [Fact]
-    public static void TestRandomness()
+    public static void TestNewGuid_Randomness()
     {
         const int Iterations = 100;
         const int GuidSize = 16;
-        byte[] random = new byte[GuidSize * Iterations];
+        var random = new byte[GuidSize * Iterations];
 
         for (int i = 0; i < Iterations; i++)
         {
             // Get a new Guid
-            Guid g = Guid.NewGuid();
-            byte[] bytes = g.ToByteArray();
+            Guid guid = Guid.NewGuid();
+            byte[] bytes = guid.ToByteArray();
```

Comment:
Nit: it's ok for now, but in the future we don't need to change things like this.  I know you said you got some feedback from someone in the past stating otherwise, but it's not necessary.  Especially when it's declared on one line and then just used immediately on the next, it's quite common and acceptable to use a single character variable name.